watchman.h

```diff
@@ -522,6 +522,9 @@ struct watchman_root {
    * of the suffix index.  Linkage via suffix_next */
   w_ht_t *suffixes;
 
+  /* collection of symlink targets that we try to watch */
+  struct watchman_pending_collection pending_symlink_targets;
```

Comment:
comment here to indicate that it is only safe to access this from the IO thread, either for reads or writes; it is not protected by the root lock.